com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -479,7 +481,10 @@ static void OnAdditionalProbesBakeCompleted()
                     cell.validity[i] = validity[j];
                 }
 
-                probeRefVolume.cells[cell.index] = cell;
+                cell.indexChunkCount = probeRefVolume.GetNumberOfBricksAtSubdiv(cell, out var minValidLocalIdxAtMaxRes, out var sizeOfValidIndicesAtMaxRes);
```

Comment:
This has the side effect of assigning the chunks in the index, we call this during LoadCell, so calling this here like this will allocate indices when it really shouldn't

Reply:
If we need this here, we need to call a version without the side-effect.

Reply:
Hm, I don't see the side effect you are talking about. Aren't you talking about GetCellIndexUpdate ?

Reply:
Yes, I need to drink my coffee :D The VS find failed me